bundles/org.openhab.binding.broadlink/src/main/java/org/openhab/binding/broadlink/config/BroadlinkDeviceConfiguration.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.broadlink.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Device configuration for the supported Broadlink devices.
+ *
+ * @author John Marshall/Cato Sognen - Initial contribution
+ */
+@NonNullByDefault
+public class BroadlinkDeviceConfiguration {
+    private String ipAddress;
+    private boolean staticIp;
+    private int port;
+    private String macAddress;
+    private int pollingInterval;
+    private String mapFilename;
+    private boolean ignoreFailedUpdates;
+    private int deviceType;
+
+    public BroadlinkDeviceConfiguration() {
+        ipAddress = "";
+        staticIp = true;
+        macAddress = "";
+        pollingInterval = 30;
+        mapFilename = "broadlink.map";
+        ignoreFailedUpdates = false;
+    }
+
+    public String getIpAddress() {
+        return ipAddress;
+    }
+
+    public void setIpAddress(String ipAddress) {
+        this.ipAddress = ipAddress;
+    }
+
+    public boolean isStaticIp() {
+        return staticIp;
+    }
+
+    public void setStaticIp(boolean staticIp) {
+        this.staticIp = staticIp;
+    }
+
+    public void setPort(int port) {
+        this.port = port;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public void setMacAddress(String macAddress) {
+        this.macAddress = macAddress;
+    }
+
+    public byte[] getMacAddress() {
+        byte configMac[] = new byte[6];
+        String elements[] = macAddress.split(":");
+        for (int i = 0; i < 6; i++) {
+            String element = elements[i];
+            configMac[i] = (byte) Integer.parseInt(element, 16);
+        }
+
+        return configMac;
+    }
```

Comment:
This gets called many times and will be a costly process parsing a string over and over again when the string is fixed and does not change. Suggest you do this once and store the byte[], then each time you need it you dont need to repeat the parsing and take more power out of the openHAB server.